workflow/kfdrc-jointgenotyping-refinement-workflow.cwl

```diff
@@ -0,0 +1,321 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc-jointgenotyping-refinement-workflow
+label: Kids First DRC Joint Genotyping Workflow
+doc: |
+  # Kids First DRC Joint Genotyping Workflow
+
+  ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
+
+  Kids First Data Resource Center Joint Genotyping Workflow (cram-to-deNovoGVCF). Cohort sample variant calling and genotype refinement.
+
+  Using existing gVCFs, likely from GATK Haplotype Caller, we follow this workflow: [Germline short variant discovery (SNPs + Indels)](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145), to create family joint calling and joint trios (typically mother-father-child) variant calls. Peddy is run to raise any potential issues in family relation definitions and sex assignment.
+
+  ### Runtime Estimates
+  - Single 5 GB gVCF Input: 90 Minutes & $2.25
+  - Trio of 6 GB gVCFs Input: 240 Minutes & $3.25 
+
+  ### Tips To Run:
+
+  1. inputs vcf files are the gVCF files from GATK Haplotype Caller, need to have the index **.tbi** files copy to the same project too.
+
+  2. ped file in the input shows the family relationship between samples, the format should be the same as in GATK website [link](https://gatkforums.broadinstitute.org/gatk/discussion/7696/pedigree-ped-files), the Individual ID, Paternal ID and Maternal ID must be the same as in the inputs vcf files header.
+
+  3. Here we recommend to use GRCh38 as reference genome to do the analysis, positions in gVCF should be GRCh38 too.
+
+  4. Reference locations:
+      - https://console.cloud.google.com/storage/browser/broad-references/hg38/v0/
```

Comment:
Needs an update -I I think the align workflow has an updated link

Reply:
Indeed it did. Updated.